includes/ucf-spotlight-common.php

```diff
@@ -13,162 +13,152 @@ public function enqueue_styles() {
 		}
 
 		public static function display_spotlight( $attr ) {
-			$output = ucf_spotlight_display( $attr );
+			$output = self::ucf_spotlight_display( $attr );
 			return apply_filters( 'ucf_spotlight_display', $output );
 		}
-	}
-
-	add_action( 'wp_enqueue_scripts', array( 'UCF_Spotlight_Common', 'enqueue_styles' ), 99 );
-}
 
-/**
-* Returns the spotlight HTML to be displayed on the page.
-* @author RJ Bruneel
-* @since 1.0.0
-* @param $attr string | slug of the spotlight post type and one of three layouts.
-* @return String
-**/
-if ( ! function_exists( 'ucf_spotlight_display' ) ) {
-	function ucf_spotlight_display( $attr ) {
-		// get post with $slug
-		if( !empty( $attr['slug'] ) ) {
+		/**
+		* Returns the spotlight HTML to be displayed on the page.
+		* @author RJ Bruneel
+		* @since 1.0.0
+		* @param $attr string | slug of the spotlight post type and one of three layouts.
+		* @return String
+		**/
+		function ucf_spotlight_display( $attr ) {
```

Comment:
Updated, I set them to private as well. https://github.com/UCF/UCF-Spotlights-Plugin/commit/6badba4784cc228e0e4ec6ce4c80c49f25dd299a